### PR TITLE
Clean up save()

### PR DESCRIPTION
**Pull request by snacsnoc**
In aa7b013 chdir() is not needed, but please correct me if I have missed core functionality. If it is needed, `_def_check_config()` in main.py will throw a sys.exit(). Perhaps for another PR, checks can be added if the file doesn't exist and perform a chdir() if needed.

For 16ba8ad I cleaned this code up to a be a bit more readable.

### CODE REVIEW

**Comment by ksawery29** [comment]
Thanks for the pull request! I look into this in a few minutes.

File: src/sushipy/main.py [hunk 1]
```diff
@@ -44,7 +44,6 @@ def _extend_check(self) -> None:
                 obj.install()
 
     def __init__(self) -> None:
-        chdir(getcwd())  # todo: do we really need that?
```
Comment (ksawery29) [inline src/sushipy/main.py:47]: Yeah, it isn't needed anymore. I added this because I had some problems with my environment.

**Review by ksawery29 — APPROVED** [review]
Looks good to me!

**Comment by snacsnoc** [comment]
Thanks for merging @ksawery29! Feel free to add more # todo's haha, this is a very interesting project

**Comment by ksawery29** [comment]
Thanks, I will haha! But currently I am focusing on the website and some other things

**Pull request merged**